Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas.git
-  revision: 933253dc37dc20c51efe962418c59f4d8bf8c75b
-  tag: v0.1.5
+  revision: 7df6ac728db24326bf32b55afb99c8607296ef85
```

Comment:
Heads up I had to re-push tag 0.1.6 as something was missing from the schema.
Doing another `bundle update laa-criminal-legal-aid-schemas` should hopefully bring the new `sha`.